circuitbreakerrpc/buf.lock

```diff
@@ -4,4 +4,4 @@ deps:
   - remote: buf.build
     owner: googleapis
     repository: googleapis
-    commit: c0ec788bbbb747fca594a1e2347edd4e
+    commit: 28151c0d0a1641bf938a7672c500e01d
```

Comment:
Do you know why this happened? I thought with all protos build in docker, it should be stable? Perhaps something isn't pinned.

Reply:
I just ran `make` in `circuitbreakerrpc` to generate? Would need to investigate further, but I re-ran to be sure and it definitely wants to update this commit.

Reply:
Ah, I see I didn't pin the dep buf.yaml (https://buf.build/docs/configuration/v1/buf-yaml). Seems fine to upgrade here. Maybe separate commit if you want to go for absolute purity.